webapp/src/components/kanban/kanban.test.tsx

```diff
@@ -88,6 +89,12 @@ describe('src/component/kanban/kanban', () => {
                 board_id_1: {userId: 'user_id_1', schemeAdmin: true},
             },
         },
+        views: {
+            views: {
+                boardView: activeView,
+            },
+            current: 'boardView',
+        },
```

Comment:
Add tests that will verify `addCardFromTemplate` gets called as appropriate. Similar to 
`test('return kanban and click on New', () => {`

Reply:
Sure